scripts/set-latest-release.ts

```diff
@@ -70,16 +72,12 @@ async function main() {
 async function withTempDir(
   cb: (tempDir: string) => Promise<void>
 ): Promise<void> {
-  const tempDir = await fs.promises.mkdtemp("radicle-dev-set-latest-release");
+  const tempDir = await fs.promises.mkdtemp(
+    path.join(os.tmpdir(), "radicle-dev-set-latest-release-")
```

Comment:
```suggestion
    path.join(os.tmpdir(), "radicle-dev-set-latest-release")
```

is that `-` intentional?

Reply:
Yes. The base string will be suffixed with some random string. To make it look nicer I added the dash.